config/default/user.role.site_manager.yml

```diff
@@ -188,6 +188,7 @@ permissions:
   - 'edit terms in hs_person_staff_type'
   - 'edit terms in hs_person_student_type'
   - 'edit terms in hs_publication_type'
+  - 'grant content access simple'
```

Comment:
What exactly does this permissions allow and could it be renamed to fit into the "access, create, edit, delete, etc.," categories used for other permissions?

Reply:
Hi @joegl, welcome back! This permission grants the ability for the site manager to see the new "Access and Permissions" details pane. See screenshot below. I copied the format from the Content Access module which uses `grant content access` as one of their permissions. We can change it if you wish as this module is still not widely used yet (Drupal.org says it's got one install and that may be Stanford?). If we changed it, I would say "access" so it would read `access content access simple` if that works for you. Let me know your preference. Thanks!

![Screenshot 2025-01-27 at 3 33 08 PM](https://github.com/user-attachments/assets/1c6cb35e-62a0-422b-91d6-951768f8a989)

Reply:
I agree with @dalin- on changing this to `access content access simple`